mlir/utils/jenkins/Jenkinsfile

```diff
@@ -213,7 +210,12 @@ pipeline {
                             }
                         }
                         stage('Performance Test') {
+                            environment {
+                                // Make libMIOpen.so accessible to the test driver
+                                LD_LIBRARY_PATH="${WORKSPACE}/MIOpen/build/lib:$LD_LIBRARY_PATH:"
```

Comment:
I don't think we need the trailing colon - though that might've been my screwup earlier.